components/UI/iconsComponents/icons/doneFilledIcon.tsx

```diff
@@ -1,14 +1,18 @@
 import React, { FunctionComponent } from "react";
 
-const DoneFilledIcon: FunctionComponent<IconProps> = ({ width, color }) => {
+const DoneFilledIcon: FunctionComponent<IconProps> = ({
+  width,
+  color,
+  filled,
+}) => {
   return (
     <svg
       xmlns="http://www.w3.org/2000/svg"
       width={width}
       height={width}
       viewBox="0 0 16 17"
     >
-      <circle cx="8" cy="8.5" r="7" fill="#402D28" />
+      <circle cx="8" cy="8.5" r="7" fill={filled} />
```

Comment:
Isn't it the `color` props that you need to use ?

Reply:
the `color` is used below in line 18. This icon needs 2 colors that's why I had to add a second variable. I think for now the component `modalProfilePic` is the only one using this icon so I could remove the `filled` variable and add it "en dur" in `doneFilledIcon` but if in the future we need it with a different color we'll face the same issue.

Reply:
You can rename the props something like `secondColor` ?